components/UserAvatar.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+const StyledAvatarImage = styled(Image)`
+  position: absolute;
+  left: 2rem;
+  top: 30rem;
+`;
+
+export default function UserAvatar({ imageSrc }) {
+  return (
+    <>
```

Comment:
If we are only returning StyleAvatarImage from this component then i don't think we need the React Fragment.